src/pudl/etl.py

```diff
@@ -370,6 +350,35 @@ def _etl_glue(glue_settings: GlueSettings) -> dict[str, pd.DataFrame]:
     return glue_dfs
 
 
+def _read_static_encoded_tables(
```

Comment:
I guess if we're going to accumulate static tables we really need to standardize an interface for they're all stored and accessed so we don't have a proliferation of special cases like this.